cli/cmd/pro.go

```diff
@@ -232,6 +233,74 @@ var centralInstallCmd = &cobra.Command{
 	},
 }
 
+var migrateCmd = &cobra.Command{
+	Use:   "migrate",
```

Comment:
how about:

```suggestion
	Use:   "activate",
```

I think "migrate" can be a bit less clear in this context.